tests/python/release_checklist/check_overrides_2d.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import annotations
+
+import os
+from argparse import Namespace
+from uuid import uuid4
+
+import rerun as rr
+import rerun.blueprint as rrb
+
+README = """
+# 2D views: Overrides, logged values, defaults, fallbacks
+
+This checks that component and visualizer overrides behave as expected for 2D views.
+
+---
+
+* Start with view **1)**.
+    * Double-click any of the arrows to select the arrow batch.
+    * Remove the extra `Points2D` visualizer.
+    * Remove all extra component overrides on the remaining `Arrows2D` visualizer.
+    * You should end up with the exact same view as the one on the right of view **1)**, which itself should look exactly like view **2)**.
+
+For reference, here is what view **1)** should look like:\
+![View 1](https://static.rerun.io/check_overrides_2d_view_1/5454441272b0abee8dbc8d7a342ecfc4036cbe9b/480w.png)
+
+---
+
+* Move over to view **2)**.
+    * Double-click any of the arrows to select the arrow batch.
+    * Remove all extra component defaults on the `Arrows2D` visualizer.
+    * You should end up with the exact same view as the one on the right of view **2)**, which itself should look exactly like view **3)**..
+
+For reference, here is what view **2)** should look like:\
+![View 2](https://static.rerun.io/check_overrides_2d_view_2/2d6f9d0cba38b3697114c4108d5d99fc5b0fd174/480w.png)
```

Comment:
as above. Well I guess you could also bundle these at the top as the first action: I.e. tell the tester to make sure that the views look like on the screenshots